hrm-domain/hrm-service/src/contact/contact-routes-v0.ts

```diff
@@ -72,6 +73,21 @@ contactsRouter.put('/:contactId/connectToCase', publicEndpoint, async (req, res)
   }
 });
 
+contactsRouter.delete('/:contactId/removeFromCase', publicEndpoint, async (req, res) => {
```

Comment:
This should be the same URL as the connect method: 
```suggestion
contactsRouter.delete('/:contactId/connectToCase', publicEndpoint, async (req, res) => {
```

Reply:
@stephenhand  - Service tests have been added and feedback implemented.